tests/inspection/test_info.py

```diff
@@ -225,3 +227,20 @@ def test_info_prefer_poetry_config_over_egg_info():
         FIXTURE_DIR_INSPECTIONS / "demo_with_obsolete_egg_info"
     )
     demo_check_info(info)
+
+
+def test_info_public_version():
```

Comment:
I'd like to see these two combined into a parameterized test, and then this should be good to go.

Reply:
`<mickey-mouse-voice>`I got three in one blow!`</mickey-mouse-voice>`